tests/SwaggerProvider.Tests/Schema.Parser.Tests.fs

```diff
@@ -62,7 +62,7 @@ let ``Schema parse of PetStore.Swagger.json sample (online)`` () =
 
 // Test that provider can parse real-word Swagger 2.0 schemas
 // https://github.com/APIs-guru/api-models/blob/master/API.md
-type ApisGuru = FSharp.Data.JsonProvider<"http://apis-guru.github.io/api-models/api/v1/list.json">
+type ApisGuru = FSharp.Data.JsonProvider<"http://api.apis.guru/v2/list.json">
```

Comment:
@IvanGoncharov  Could you plz change url to `httpS://` here? Something strange happened on `Mono`
Thanks